scripts/west_commands/runners/blackmagicprobe.py

```diff
@@ -5,11 +5,59 @@
 '''Runner for flashing with Black Magic Probe.'''
 # https://github.com/blacksphere/blackmagic/wiki
 
+import os
 import signal
+import sys
+import glob
 from pathlib import Path
 
 from runners.core import ZephyrBinaryRunner, RunnerCaps
 
+# Default path for linux, based on the project udev.rules file.
+DEFAULT_LINUX_BMP_PATH = '/dev/ttyBmpGdb'
+
+LINUX_SERIAL_GLOB = '/dev/ttyACM*'
+DARWIN_SERIAL_GLOB = '/dev/cu.usbmodem*'
+
+def BlackMagicProbeGDBSerial(port):
```

Comment:
camel case is for class names, not function names

Reply:
Sorted.